packages/checkout/widgets-lib/src/context/view-context/BridgeViewContextTypes.ts

```diff
@@ -59,3 +66,18 @@ interface BridgeClaimWithdrawal extends ViewType {
   type: BridgeWidgetViews.CLAIM_WITHDRAWAL,
   transaction: Transaction
 }
+
+interface BridgeClaimWithdrawalInProgress extends ViewType {
+  type: BridgeWidgetViews.CLAIM_WITHDRAWAL_IN_PROGRESS,
+  transactionResponse: TransactionResponse;
+}
+
+export interface BridgeClaimWithdrawalSuccess extends ViewType {
+  type: BridgeWidgetViews.CLAIM_WITHDRAWAL_SUCCESS,
+  transactionHash: string;
+}
+
+interface BridgeClaimWithdrawalFailure extends ViewType {
```

Comment:
are you passing the transactionHash through if there is a failure? To be added to the event?